### PR TITLE
chore: use Apache 2 license instead of GPLv3

### PR DESCRIPTION
**Pull request by albemarle**

### CODE REVIEW

**Comment by nedbat** [comment]
Great! Also update setup.py: https://github.com/edx/edx-drf-extensions/blob/master/setup.py#L15-L20

**Comment by nedbat** [comment]
And the readme: https://github.com/edx/edx-drf-extensions/blame/master/README.rst#L33

**Comment by albemarle** [comment]
@nedbat Updated those places (though it seems to me we could take it out of the README...).

File: setup.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
     author='edX',
     author_email='oscm@edx.org',
     url='https://github.com/edx/edx-drf-extensions',
-    license='AGPL',
+    license='Apache 2.0',
     classifiers=[
         'Development Status :: 5 - Production/Stable',
         'Environment :: Web Environment',
```
Comment (nedbat) [inline setup.py]: Sorry to be persnickety, but two lines down, it should say:
```
    'License :: OSI Approved :: Apache Software License',
```

Comment (albemarle) [reply setup.py]: All set!

**Pull request merged**